btk/draw_blends.py

```diff
@@ -182,6 +185,17 @@ def __init__(
         self.channels_last = channels_last
         self.save_path = save_path
 
+        if rng is None:
+            self.rng = np.random.default_rng()
+        elif isinstance(rng, int):
+            self.rng = np.random.default_rng(rng)
+        else:
+            try:
+                rng.random()
+            except AttributeError:
+                raise AttributeError("The random generator you provided is invalid.")
```

Comment:
so here if I give `rng = numpy.random` this will work but good luck for reproducibility.
I guess we should restrict only the cases to
- a seed is provided
- a default seed is used